src/Page/Programmer/index.js

```diff
@@ -34,7 +34,7 @@ function Programmer() {
                 className="btn btn-primary text-white w-100 my-1 fs-sm-1.2  regular-btn"
                 onClick={(e) => {
                   e.preventDefault();
-                  window.open("https://admissions.navgurukul.org/", "_blank");
+                  window.open("https://www.merakilearn.org/admission", "_blank");
```

Comment:
Could we create a constant like `ADMISSIONS_URL` in a config file and assign it to `https://www.merakilearn.org/admission` so we could import it instead of hard-coding it? This way, if the URL were to ever change again in the future, it could be updated in one central place without having to find and replace all of the occurrences of it in source code.